internal/controller/driver_controller.go

```diff
@@ -946,6 +970,10 @@ func (r *driverReconcile) reconcileNodePluginDeamonSet() error {
 								})
 							}
 						}
+						// CSI Logrotate Container
+						if r.driver.Spec.Log.Rotation != nil {
+							containers = append(containers, r.logrRotateContainer(imagePullPolicy, daemonSet.Name, r.driver.Spec.NodePlugin.Resources.LogRotator))
+						}
```

Comment:
Please inline the log rotate container definition

Reply:
this isn't more readable, what is your suggestion can you elaborate